### PR TITLE
travis: drop Symfony 2.3, 3.0 and add Symfony 3.2, 3.3

### PR DESCRIPTION
**Pull request by TomasVotruba**
2.3 is [past EOL](https://symfony.com/roadmap?version=2.3#checker) and 3.3 might be nice to test

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ php:
   - '7.0'
 
 env:
-  - SYMFONY_VERSION="2.3.*"
   - SYMFONY_VERSION="2.7.*"
   - SYMFONY_VERSION="3.0.*"
```
Comment (dmaicher) [inline .travis.yml]: actually `3.0.*` is also not maintained anymore :smile: We can switch to `3.2.*`?

Comment (TomasVotruba) [inline .travis.yml]: Good point :rocket:

**Comment by dmaicher** [comment]
Thanks! :-1:

**Comment by TomasVotruba** [comment]
You're welcome :

**Pull request merged**